js/src/wp-seo-post-scraper.js

```diff
@@ -1,9 +1,13 @@
-/* global YoastSEO, tinyMCE, wp, ajaxurl, wpseoPostScraperL10n, YoastShortcodePlugin, YoastReplaceVarPlugin, console */
+/* global YoastSEO, tinyMCE, ajaxurl, wpseoPostScraperL10n, YoastShortcodePlugin, YoastReplaceVarPlugin, console, require */
 (function( $ ) {
 	'use strict';
 
 	var currentKeyword = '';
 
+	// Initialize the keyword tab module.
```

Comment:
Initialize is the wrong word for this, you're defining the variable, not initializing it.